sanic/sanic.py

```diff
@@ -247,7 +243,7 @@ def converted_response_type(self, response):
     def run(self, host="127.0.0.1", port=8000, debug=False, before_start=None,
             after_start=None, before_stop=None, after_stop=None, sock=None,
             workers=1, loop=None, protocol=HttpProtocol, backlog=100,
-            stop_event=None):
+            stop_event=None, logger=None):
```

Comment:
@awiddersheim yes, I know it's irking me. It's a related to an open bug here: https://github.com/channelcat/sanic/issues/73. It could be removed until that's fixed, but it's critical, especially to testing multiprocessing -- as far as I know.